tests/unit/Checkout/Cart/Controllers/CartControllerCest.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+namespace Tests\Unit\Checkout\Cart\Controllers;
+
+use UnitTester;
+use Webkul\Checkout\Models\Cart;
+use Webkul\Shop\Http\Controllers\CartController;
+
+class CartControllerCest
+{
+    public function _before(UnitTester $I)
+    {
+    }
+
+    public function testOnWarningAddingToCart(UnitTester $I)
+    {
+        $scenarios = [
+            [
+                'result' => ['key' => 'value', 'warning' => 'Hello World. Something went wrong.'],
```

Comment:
I propose to fake the content of the warning, to cover edge cases like umlauts.

Reply:
The purpose of this unit test is to check, if a result (as type of array) contains a key "warning". The value of this key itself is not used. Therefore I think, we don't need to fake this value.

Reply:
Ah, you are right. Okay then :)

Reply:
Ah, you are right. Okay then :) Thanks.